### ISSUE TITLE
Question about object detection pre-training.

### ISSUE THREAD

**Issue by leng-yue** [OFA-Sys/OFA#261]
After viewing `detection_examples.tsv` and the data loading code, I can't find any order in the bounding boxes. If there is no order in the bounding box, how does OFA calculate the detection loss?

Thanks for your high-quality work :)

**Comment by logicwong** [comment]
@leng-yue We follow [pix2seq](https://arxiv.org/abs/2109.10852) to calculate the detection loss. In order to reduce the training difficulty, we sort the objects by the object size in the preprocessing stage.

**Comment by leng-yue** [comment]
Thanks for your clarification :)

**Comment by leng-yue** [comment]
Sorry for bothering again. While the pix2seq paper says that random ordering yields better performance, did you try it?

**Comment by logicwong** [comment]
@leng-yue We didn't try random order.

**Comment by JulioZhao97** [comment]
> @leng-yue We follow [pix2seq](https://arxiv.org/abs/2109.10852) to calculate the detection loss. In order to reduce the training difficulty, we sort the objects by the object size when processing the data.

Sorry to bother! But I find that detection loss in pix2seq is very low (~0.6) in a few epochs, but in OFA pretraining the detection loss is around 5. What's the difference? Thank you very much!
